### ISSUE TITLE
Export only work items of the current team project

### ISSUE THREAD

**Issue by shayki5** [snuziale/vsts-open-in-excel#20]
Hi,

If I create query "Query across projects" and export the results to excel, only work items of the current project (where I run the query) are exported.

I got this message:

![image](https://user-images.githubusercontent.com/26063853/46787895-7e47c080-cd41-11e8-89a8-11c191dba749.png)

Can you fix it?

Thank you!

**Comment by bsc7080hum** [comment]
I don't believe this is a bug but working as intended. Keep in mind that the process template could be different between different Team Projects, thus fields will not align properly. Microsoft now provides the ability to export to CSV which does cross projects. Excel is an update capable solution thus the restrictive behavior.

**Comment by shayki5** [comment]
Ok, make sense.
Thank you for your response!
